examples/cxx/publish_subscribe_cross_language/README.md

```diff
@@ -0,0 +1,136 @@
+# Cross-language Publish-Subscribe
+
+Before proceeding, all dependencies need to be installed. You can find
+instructions in the [C++ Examples Readme](../README.md).
+
+## Running The Example
+
+> [!CAUTION]
```

Comment:
Can you add this caution also to the C cross language example readme, except the last part - this is C++ specific.